core/src/main/java/com/exadel/aem/toolkit/bundle/lists/datasource/ItemComponentDatasource.java

```diff
@@ -42,27 +42,27 @@
 import com.adobe.granite.ui.components.ds.ValueMapResource;
 
 /**
- * Servlet that implements {@code datasource} pattern for populating a TouchUI {@code select} widget
- * with item components that have acl-item-component group in a Granite datasource
+ * Servlet that implements {@code datasource} pattern for populating a TouchUI {@code Select} widget
```

Comment:
May I advise to avoid using structure `XXX that does YYYY` everywhere in class-level and field-level javadocs. It is both redundant (we know that it's a servlet from looking ant "extends... Servlet") and may complicate the phrase (technically it turns a simple sentense into a complex sentence that has a main and an additional part, questions on comm usage araise, etc. Also issues with articles are brought in: A servlet... the servlet.. just "servlet"&.. Start with the verb, and the problem is gone.

Reply:
Reformulated